django_project/api/utils.py

```diff
@@ -12,3 +14,76 @@ def remap_dict(old_dict, transform):
         else:
             new_dict.update({k: v})
     return new_dict
+
+
+def changeset_tags(comment=None):
```

Comment:
could you put this into `OsmApiWrapper` class? because this function is just for that class